extensions/amp-story/1.0/page-advancement.js

```diff
@@ -430,10 +430,7 @@ class ManualAdvancement extends AdvancementConfig {
       dev().assertElement(event.target),
       el => {
         tagName = el.tagName.toLowerCase();
-        if (
-          tagName === 'amp-story-page-attachment' ||
-          tagName === 'amp-story-quiz'
-        ) {
+        if (el.classList.contains('i-amphtml-story-handle-tap-manually')) {
```

Comment:
Please declare this as a constant that other files can import

Additionally, should we use an attribute instead of a class? Classes are mostly reserved for styling